test/notebooks/notebook_test_utils.py

```diff
@@ -54,7 +56,7 @@ def cell_match(nb_cell, ins_tag: str) -> bool:
 
 
 def run_notebook(notebook_file: str, store_file: str, store_password: str,
-                 timeout: int = -1, hacks: Optional[List[Tuple[str, str]]] = None) -> None:
+                 timeout: int = -1, hacks: Optional[List[Tuple[str, str]]] = None) -> nbformat.NotebookNode:
```

Comment:
I thought a bit about when I saw your struggles with imports, maybe we should add a parameter with which you can enable the output. Instead of that we need to heavily modify the tests.